### PR TITLE
[Qt] Added utest and doc stubs

### PR DESCRIPTION
**Pull request by tmpsantos**
Foundations for adding documentation to every single public API and also unit tests for them.

### CODE REVIEW

**Comment by tmpsantos** [comment]
:eyes:  @brunoabinader

File: platform/qt/src/qmapboxgl.cpp [hunk 1]
```diff
@@ -118,6 +118,38 @@ auto fromQStringList(const QStringList &list)
 
 }
 
+/*!
+    \class QMapboxGLSettings
+    \brief The QMapboxGLSettings stores the initial configuration for a QMapboxGL.
```
Comment (brunoabinader) [inline platform/qt/src/qmapboxgl.cpp]: `The QMapboxGLSettings` 👉 `The QMapboxGLSettings class`
`for a QMapboxGL` 👉 `for QMapboxGL object`.

File: platform/qt/src/qmapboxgl.cpp [hunk 2]
```diff
@@ -118,6 +118,38 @@ auto fromQStringList(const QStringList &list)
...
+
+    \inmodule Mapbox Qt SDK
+
+    QMapboxGLSettings is used to configure a QMapboxGL at the moment of its creation.
```
Comment (brunoabinader) [inline platform/qt/src/qmapboxgl.cpp]: `to configure a QMapboxGL` 👉 `to configure QMapboxGL`

**Comment by brunoabinader** [comment]
👍 great start towards full documentation and unit tests for the Qt SDK!

**Pull request merged**